docs/roman/error_propagation/main.rst

```diff
@@ -0,0 +1,135 @@
+
+fringe 
+++++++
+For MIRI MRS (IFU) mode exposures, the SCI and ERR arrays in the science exposure
+are divided by the fringe reference image.  For details of the fringe correction, see 
+:ref:`fringe <fringe_step>`.
+
+barshadow 
++++++++++
+This step is applied only to NIRSpec MSA data for extended sources. Once the
+2-D correction array for each slit has been computed, it is applied to the
+science (SCI), error (ERR), and variance (VAR_POISSON, VAR_RNOISE, and VAR_FLAT)
+arrays of the slit.  The correction values are divided into the SCI and ERR
+arrays, and the square of the correction values are divided into the variance 
+arrays.   For details of the bar shadow correction, see
+:ref:`barshadow <barshadow_step>`.
+
+pathloss
+++++++++
+The ``pathloss`` step corrects NIRSpec and NIRISS SOSS data for various types of
+light losses. The correction factors are divided into the SCI and ERR arrays of
+the science data, and the square of the correction values are divided into the
+variance arrays. For details of this step, see :ref:`pathloss <pathloss_step>`.
+
+photom
+++++++ 
+The calibration information for the ``photom`` step includes a scalar flux conversion
+constant, as well as optional arrays of wavelength and relative response (as a
+function of wavelength). The combination of the scalar conversion factor and any 2-D
+response values is applied to the science data, including the SCI and ERR arrays,
+as well as the variance (VAR_POISSON, VAR_RNOISE, and VAR_FLAT) arrays. The flux
+calibration values are multiplied into the science exposure SCI and ERR arrays,
+and the square of the calibration values is multiplied into all variance arrays.
+For details of the photom correction, see :ref:`photom <photom_step>`.
+
+Stage 3 pipelines
+-----------------
+Stage 3 pipelines perform operations that work with multiple exposures and in
+most cases produce some kind of combined product.  The operations in these
+pipelines that either use or modify variance/error arrays that are propagated 
+through the pipeline are ``outlier_detection`` and ``wfs_combine``.
+
+outlier_detection
++++++++++++++++++
+The ``outlier_detection`` step is used in all Stage 3 pipelines.  It uses the ERR array to
+make a local noise model, based on the readnoise and calibration errors of earlier 
+steps in the pipeline. This step does not modify the ERR array or any of the VAR
+arrays.
+
+wfs_combine
++++++++++++
+The ``wfs_combine`` step is only applied in the Stage 3 Wavefront Sensing and Control
+(calwebb_wfs-image3) pipeline for dithered pairs of WFS&C exposures.
+This step can modify variance/error arrays, but only if the optional
+"do_refine" parameter is set to True (the default is False). In this
+case the algorithm to refine image offsets will be used and the ERR array values will be
+altered on output, using a combination of the input image errors.
+See the step documentation at :ref:`wfs_combine <wfs_combine_step>` for
+more details.
```

Comment:
Is this a placeholder file for now? The contents are still JWST-specific.

Reply:
Yes, there is much in these files that is JWST-specific, for the immediate goal of getting  readthedocs to build successfully.

Reply:
Okay, approving to get the rtds to build